release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -4039,4 +4039,23 @@ For more information about logging levels for MetalLB, see xref:../networking/me
 [id="ocp-4-10-60-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-61"]
+=== RHSA-2023:3363 - {product-title} 4.10.61 bug fix and security update
+
+Issued: 2023-06-07
+
+{product-title} release 4.10.61 is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:3363[RHSA-2023:3363] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3362[RHSA-2023:3362] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.10.61 --pullspecs
+----
+
+[id="ocp-4-10-61-upgrading"]
+==== Updating
+
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
```

Comment:
Update title as per a previous comment.

Reply:
@dfitzmau Do you it should say:
==== Updating a cluster using the CLI  ?

Reply:
Hi @cbippley . The title of this section is OK, but the link anchor text does not match the source heading.

Updating a cluster within a minor version by using the CLI > Updating a cluster using the CLI